lib/src/editor/ui/editor_tabs.dart

```diff
@@ -91,35 +94,42 @@ class _CoreContentTabsState extends State<CoreContentTabs> {
             ),
           ),
           const TabsVerticalDivider(),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().zoomIn,
-            icon: FeatherIcons.zoomIn,
-          ),
-          const SizedBox(width: 8),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().zoomOut,
-            icon: FeatherIcons.zoomOut,
-          ),
-          const SizedBox(width: 8),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().resetZoom,
-            icon: FeatherIcons.refreshCcw,
-          ),
-          StyledTextButton(
-            onPressed: context.read<GridProvider>().toggleGrid,
-            icon: FeatherIcons.grid,
-          ),
-          const TabsVerticalDivider(),
-          const SizedBox(width: 8),
+          context.read<TabProvider>().tab == FlutterBookTab.canvas
```

Comment:
- Always use `if` statements for conditionally rendering widgets in a list rather than ternary statements, they are much easier to read 
- this is a lot of conditionally rendered code, I'd recommend splitting it out into a small private widget in this file